Entry.php

```diff
@@ -16,6 +16,18 @@ function getTitle()
 
         function getDescription()
         {
+            $body = $this->body;
+            if (!empty($this->inreplyto)) {
+                if (is_array($this->inreplyto)) {
```

Comment:
Can you use a temporary array variable to avoid repeating the "<a href..." boilerplate

```
  $x = (is_array($x)) ? ($x) : (array($x));
```